plugins/kubernetes/app/models/kubernetes/job_executor.rb

```diff
@@ -0,0 +1,96 @@
+# Execute a job that can build/push Docker images
+# and write progress to the local job output.
+module Kubernetes
+  class JobExecutor
```

Comment:
Call this something like `BuildJobExecutor`, since it's specifically for executing a job that builds Docker images, rather something that executes jobs generically.